packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -0,0 +1,352 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/physics.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/scheduler.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+import 'gesture_detector.dart';
+import 'scroll_metrics.dart';
+import 'scroll_notification.dart';
+import 'scroll_position.dart';
+import 'ticker_provider.dart';
+
+abstract class ScrollActivityDelegate {
```

Comment:
I have mixed feelings about the existence of this interface since we just got rid of it last week, but I'm willing to see where it takes us.